Makefile

```diff
@@ -13,6 +13,7 @@ GO_BUILD_PACKAGES :=$(GO_PACKAGES)
 GO_BUILD_PACKAGES_EXPANDED :=$(GO_BUILD_PACKAGES)
 # LDFLAGS are not needed for dummy builds (saving time on calling git commands)
 GO_LD_FLAGS:=
+CONTROLLER_GEN_VERSION :=v0.2.5
```

Comment:
@tnozicka as a follow-up we might want to use this as the default in build-machinery